internal/statesync/reactor.go

```diff
@@ -1013,9 +1013,11 @@ func (r *Reactor) waitForEnoughPeers(ctx context.Context, numPeers int) error {
 		iter++
 		select {
 		case <-ctx.Done():
-			return fmt.Errorf("operation canceled while waiting for peers after %s", time.Since(startAt))
+			return fmt.Errorf("operation canceled while waiting for peers after %2fs [%d/%d]",
```

Comment:
Did you mean `%.2f` here and below? 

If the goal is just to round to seconds, another option is to use `%v` with `time.Since(startAt).Round(time.Second)`, YMMV

Reply:
yeah, I thought like "fractional seconds but like, truncated?)